fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -36,9 +36,20 @@ set(FastRTPS_FOUND FALSE)
 find_path(FastRTPS_INCLUDE_DIR
   NAMES fastrtps/)
 
+find_library(FastRTPS_LIBRARY
+    NAMES fastrtps-1.4 fastrtps)
```

Comment:
Does this imply that every time FastRTPS releases a new minor version the Windows builds will fail and this line needs to be updated to make it work again?